casanovo/denovo/model.py

```
@@ -1185,10 +1184,12 @@ def predict_step(
 
         return predictions
 
-    def on_train_epoch_end(self) -> None:
+    def on_train_batch_end(self, outputs, batch, batch_idx) -> None:
```

Ruff (0.15.5)

[warning] Unused method argument: `outputs`

(ARG002)